h/views/api/exceptions.py

```diff
@@ -16,11 +16,12 @@
 from h.exceptions import APIError
 from h.schemas import ValidationError
 from h.util.view import handle_exception, json_view
+from h.views.api.config import cors_policy
 
 
 # Within the API, render a JSON 403/404 message.
-@forbidden_view_config(path_info='/api/', renderer='json')
-@notfound_view_config(path_info='/api/', renderer='json')
+@forbidden_view_config(path_info='/api/', renderer='json', decorator=cors_policy)
```

Comment:
Musing...I wonder if this `exceptions` module warrants a comment somewhere to help future-devs from forgetting to add the decorator when necessary and/or explaining what it does. I know it's done well in `h.views.api.config` but that's not readily visible here.